web/src/components/organisms/Project/index.tsx

```diff
@@ -78,6 +78,7 @@ const Project: React.FC = () => {
 const PaddedContent = styled(Content)`
   margin: 16px;
   background-color: #fff;
+  min-height: 100%;
```

Comment:
Do we want the content to be able to be more than 100%? If not, 
```suggestion
  height: 100%;
```

Reply:
done.